src/components/captions-control/captions-control-mini.tsx

```diff
@@ -0,0 +1,112 @@
+import {createPortal, useEffect} from 'preact/compat';
```

Comment:
huh. where did the "/compat" come from ?

Reply:
hmm because of the `createPortal`

Reply:
I mean, we usually import this from "preact"
How come this came out different ?
"compat" has createPortal and normal preact doesn't ? why ?

Reply:
I'm not sure, I took the same import we do in closed captions component..

Reply:
@SivanA-Kaltura wdym "normal preact"?
so we have `preact/hooks` which exports the hooks (`useEffect`, `useMemo`, etc...) and then there's `preact/compat` which exports the `createPortal` and apparently also the hooks from `preact/hooks`, see the code line below from `preact/compat`:
`export import useEffect = _hooks.useEffect;`

To sum up:
- `createPortal` can be imported **only** from `preact/compat`
- while the hooks can be imported from `preact/compat` **and** `preact/hooks`